vocabularies/Common.xml

```diff
@@ -1454,6 +1454,105 @@ If the request succeeds the response will contain a JSON body of `Content-Type:
+        </ReturnType>
+      </Function>
+      <ComplexType Name="RecommendationsTargetsType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="URLs" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.IsURL" />
+          <Annotation Term="Core.Description" String="URLs (relative to the request URL) that address entities, properties or action/function parameters for which recommendations shall be retrieved" />
+          <Annotation Term="Core.LongDescription">
+            <String>The request `SalesOrder('A')/ns.GetRecommendations?Targets={"URLs":["","Items(10)"]}`
+            retrieves recommendations for the entities `SalesOrder('A')` and `SalesOrder('A')/Items(10)`.</String>
+          </Annotation>
+        </Property>
+      </ComplexType>
+      <ComplexType Name="PropertyRecommendationType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="AIRecommendedFieldPath" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.IsURL" />
+          <Annotation Term="Core.Description" String="URL (relative to the URL of the [`GetRecommendations`](#GetRecommendations) function request) addressing a property [OData-URL, section 4.6] that is targeted by this recommendation" />
+          <Annotation Term="Core.LongDescription">
+            <String>The request `SalesOrder('A')/ns.GetRecommendations?Targets=...`
+            might retrieve a recommendation with `"AIRecommendedFieldPath": "Items(10)/Product"`.</String>
+          </Annotation>
+        </Property>
+        <Property Name="AIRecommendedFieldValue" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Recommended value, converted to string" />
+          <Annotation Term="Common.Text" Path="text" />
+        </Property>
+        <Property Name="AIRecommendedFieldDescription" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.Description" String="Description of the recommended value" />
+        </Property>
+        <Property Name="AIAltvRecmddFldVals" Type="Collection(Common.AlternativeRecommendationType)" Nullable="false">
+          <Annotation Term="Core.Description" String="A list of alternative values, sorted by confidence score in descending order" />
+          <Annotation Term="Core.LongDescription">
+            <String>If a value is recommended via property `AIRecommendedFieldValue`, it must be the first entry in this list.</String>
+          </Annotation>
+        </Property>
+      </ComplexType>
+      <ComplexType Name="AlternativeRecommendationType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="AIRecommendedFieldValue" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Alternatively recommended value, converted to string" />
+        </Property>
+        <Property Name="AIRecommendedFieldScore" Type="Edm.Decimal" Scale="variable" Nullable="true">
```

Comment:
```suggestion
        <Property Name="RecommendedFieldValue" Type="Edm.String" Nullable="false">
          <Annotation Term="Core.Description" String="Alternatively recommended value, converted to string" />
        </Property>
        <Property Name="RecommendedFieldScore" Type="Edm.Decimal" Scale="variable" Nullable="true">
```

Reply:
We now go into the other direction: Add the AI prefix everywhere.